### PR TITLE
VB-752: Add initial view by visits layout

### PR DESCRIPTION
**Pull request by psoleckimoj**

### CODE REVIEW

File: server/config.ts [hunk 1]
```diff
@@ -4,7 +4,11 @@ const production = process.env.NODE_ENV === 'production'
 
 function get<T>(name: string, fallback: T, options = { requireInProduction: false }): T | string {
   if (process.env[name]) {
-    return process.env[name]
+    if (production) {
+      return process.env[name]
+    }
+
+    return process.env[name].replace(/\\/g, '')
```
Comment (tpmcgowan) [inline server/config.ts:11]: Is this the fix for `.env` issues you had?

**Review by tpmcgowan — APPROVED** [review]
LGTM!

File: server/config.ts (hunk 1 above)
Comment (psoleckimoj) [reply server/config.ts:11]: Yep, I left production as is, although maybe I need to work out when we're running locally?

Comment (tpmcgowan) [reply server/config.ts:11]: 👍 perhaps you could just have something else in the `.env` file, like `MESSY_TOKEN=true`!

**Pull request merged**